tests/console/system_state.pm

```diff
@@ -19,11 +28,14 @@ use utils;
 use Utils::Backends 'use_ssh_serial_console';
 use strict;
 use warnings;
+use opensusebasetest;
 
 sub run {
     check_var("BACKEND", "ipmi") ? use_ssh_serial_console : select_console 'root-console';
-    script_run "ps axf > /tmp/psaxf.log";
-    script_run "cat /proc/loadavg > /tmp/loadavg_consoletest_setup.txt";
+    script_run "mkdir /tmp/system_state";
+    script_run "ps axf > /tmp/system_state/psaxf.log";
+    script_run "cat /proc/loadavg > /tmp/system_state/loadavg_consoletest_setup.txt";
+    opensusebasetest->tar_and_upload_log('/tmp/system_state', '/tmp/stats_during_installation.tar.bz2');
```

Comment:
As `tar_and_upload_log` uses `self` I would propose to add `my ($self) = @_;` and call it same way as it's expected.
Otherwise, I don't understand why you want to send `opensusebasetest` and not base class at least.
So anyway, let's do it with `$self`, if you don't object.

Reply:
correct. my mistake. $self is more proper and correct here